src/components/PersonalSettings.vue

```diff
@@ -0,0 +1,264 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<div id="richdocuments" class="section">
+		<h2>
+			{{ t('richdocuments', 'Nextcloud Office') }}
+		</h2>
+
+		<span id="documents-admin-msg" class="msg" />
+
+		<!-- Template folder selection -->
+		<p>
+			<label for="templateInputField">
+				{{ t('richdocuments', 'Select a template directory') }}
+			</label><br>
+			<input id="templateInputField"
+				type="text"
+				name="templateInputField"
+				:value="templateFolder"
+				disabled>
+			<button id="templateSelectButton"
+				@click="onTemplateSelectButtonClick">
+				<span class="icon-folder"
+					:title="t('richdocuments', 'Select a personal template folder')"
+					data-toggle="tooltip" />
+			</button>
+			<button id="templateResetButton"
+				@click="resetTemplate">
+				<span class="icon-delete"
+					:title="t('richdocuments', 'Remove personal template folder')"
+					data-toggle="tooltip" />
+			</button>
```

Comment:
Please also use NcButton and [NcTextField](https://nextcloud-vue-components.netlify.app/#/Components/NcFields?id=nctextfield) as components

Reply:
Done